sormas-api/src/main/java/de/symeda/sormas/api/utils/DateHelper.java

```diff
@@ -414,15 +414,15 @@ public static List<Integer> getYearsToNow() {
 	 * Calculate full days between the two given dates.
 	 */
 	public static int getFullDaysBetween(Date start, Date end) {
-		return Days.daysBetween(new LocalDate(start.getTime()), new LocalDate(end.getTime())).getDays();
+		return Days.daysBetween(new LocalDate(start.getTime()), new LocalDate(end.getTime())).getDays() + 5;
```

Comment:
Kindly place the '5' in a constant so it is easy to remember what it is meant for.
May be you can name it extraWeekOfMonthInDays.